tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -696,6 +820,52 @@ static uint8_t bap_discover(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_SUCCESS;
 }
 
+static uint8_t bap_send(const void *cmd, uint16_t cmd_len,
+			void *rsp, uint16_t *rsp_len)
+{
+	int err;
+	const struct btp_bap_send_cmd *cp = cmd;
+	struct audio_connection *audio_conn;
+	struct audio_stream *stream;
+	struct bt_conn *conn;
+	struct net_buf *buf;
+	struct bt_conn_info conn_info;
+
+	conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT, &cp->address);
+	if (!conn) {
+		LOG_ERR("Unknown connection");
+		return BTP_STATUS_FAILED;
+	}
+
+	audio_conn = &connections[bt_conn_index(conn)];
+	(void)bt_conn_get_info(conn, &conn_info);
+
+	stream = stream_find(audio_conn, cp->ase_id);
+	if (stream == NULL) {
+		return BTP_STATUS_FAILED;
+	}
+
+	LOG_DBG("Sending data to ASE: ase_id %d dir %d len %d seq %d", cp->ase_id,
+		stream->stream.dir, cp->data_len, stream->seq_num);
```

Comment:
`dir` is also kind of irrelevant here, as it will always be `SINK` for the client. Or rather, if it isn't `SINK` it is an invalid operation